src/Autosuggest.js

```diff
@@ -18,6 +20,12 @@ export default class Autosuggest extends Component { // eslint-disable-line no-s
     id: PropTypes.string                    // Used in aria-* attributes. If multiple Autosuggest's are rendered on a page, they must have unique ids.
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (typeof nextProps.value === 'string') {
+      this.onUpdate(nextProps.value);
+    }
```

Comment:
If we verify the type here, we shouldn't fail silently. I suggest we have something like:
```
else {
  throw new Error('value must be a string. Found:', nextProps.value);
}
```